src/utils/debug.js

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable no-console */
+const isProduction = (typeof process === 'object') && (typeof process.env === 'object') && (process.env.NODE_ENV === 'production');
+const logStyles = 'background:#212121; padding:2px; border: 1px solid #646464; border-radius:3px; color:#fff';
+
+const componentNamespace = (componentName) => (componentName?.trim() ? ` [${componentName}]` : '');
```

Comment:
I think it should be _required_ to pass in a component name.

Have you encountered any cases where it shouldn't be required?

Reply:
I'm ok with either. I'll make it required.